openapi/data/fields.py

```diff
@@ -100,7 +100,7 @@ def date_time_field(required=False):
 # VALIDATORS
 
 
-def email_validator(field, value, data):
```

Comment:
why are you removing the data? it may be required for more complex validators

Reply:
There were no validators using it, so I removed. We can add it if we need in the future